cmd/kev/cmd/render.go

```diff
@@ -45,6 +47,7 @@ var renderCmd = &cobra.Command{
 			if err := fn(cmd, args); err != nil {
 				return err
 			}
+			os.Stdout.Write([]byte("\n"))
```

Comment:
I think I've seen it a few times. Could this go in one of functions?

Reply:
Yup, its used twice. For now, I can't think of an elegant way to abstract this.

Its used as a cmd separator, in a context where the cmds are unaware of how they're ordered and therefore cannot assert if a new line is required or not.

TBH, the UX still needs more love. So will definitely revisit this in the future. The `logrus` lib is a bit lacking.